src/Parser/HolyPaladin/Modules/PaladinCore/FillerLightOfTheMartyrs.js

```diff
@@ -6,25 +6,42 @@ import SpellLink from 'common/SpellLink';
 import ItemLink from 'common/ItemLink';
 
 import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
 
 import AbilityTracker from './PaladinAbilityTracker';
 import MaraadsDyingBreath from '../Items/MaraadsDyingBreath';
 
 class FillerLightOfTheMartyrs extends Module {
   static dependencies = {
+    combatants: Combatants,
     abilityTracker: AbilityTracker,
     maraadsDyingBreath: MaraadsDyingBreath,
+    spellUsable: SpellUsable,
   };
 
-  suggestions(when) {
-    const getAbility = spellId => this.abilityTracker.getAbility(spellId);
+  casts = 0;
+  inefficientCasts = 0;
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.LIGHT_OF_THE_MARTYR.id) {
+      return;
+    }
+    if (this.combatants.selected.hasBuff(SPELLS.MARAADS_DYING_BREATH_BUFF.id, event.timestamp)) {
+      // Not a filler
+      return;
+    }
 
-    const lightOfTheMartyrs = getAbility(SPELLS.LIGHT_OF_THE_MARTYR.id).casts || 0;
-    let fillerLotms = lightOfTheMartyrs;
-    if (this.maraadsDyingBreath.active) {
-      const lightOfTheDawns = getAbility(SPELLS.LIGHT_OF_DAWN_CAST.id).casts || 0;
-      fillerLotms -= lightOfTheDawns;
+    this.casts += 1;
```

Comment:
Slightly misleading: this isn't casts, this is "casts without Maraads". Maybe rename it `fillerCasts`, considering how it's used in suggestions?

Reply:
We're in the context "FillerLightOfTheMartyrs", I feel naming a properties "fillers" would be double which I feel would be detrimental.

Reply:
Clearly you should also rename the module LightOfTheMarty :smile:  ... but yeah that's a good enough explanation.

Reply:
If this becomes necessary then the class name will probably also become invalid. ;)